src/nft/components/profile/list/MarketplaceRow.tsx

```diff
@@ -267,6 +279,7 @@ export const MarketplaceRow = ({
             globalOverride={globalOverride}
             warning={warning}
             asset={asset}
+            shrink={isMobile}
```

Comment:
nit: considering this is the only time this component is being used in the codebase why not dynamically change the size of the wrapper in CSS in the component?

Reply:
good call out, updated